client/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
I managed to fix the preflight check error locally. 

1. Remove yarn.lock and nodemodules
2. Remove `eslint` and `jest` from devdependencies in react's package.json file
3. do yarn install again
4. delete .env file (also put it in .gitignore so someone doesn't accidentally commit and env file in future.)

Reply:
https://github.com/facebook/create-react-app/issues/5247

Reply:
since on client sides .env files are generally not used, because at the time of build time env variables are replaced by their actual values, https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables.

But still, it's a good practice to keep it in .gitignore.